### PR TITLE
[4.0] Label com_content

### PR DESCRIPTION
**Pull request by hardik-codes**
This PR adds a label in the tab where only editor is present in com_content. 

### Testing Instructions
Open any article in articles overview

### Expected result
A label is there in a tab where only editor is present.
Like in the case of com_category
com_categories

In case of com_contact
com_contact(3)

### Actual result
No label is there

### Documentation Changes Required
None

### CODE REVIEW

**Comment by brianteeman** [comment]
This is not correct - see #23946

**Comment by hardik-codes** [comment]
@brianteeman IMO this PR more or less does the same thing as #23946

**Comment by brianteeman** [comment]
~It does the opposite~

**Comment by hardik-codes** [comment]
> It does the opposite

Could you please elaborate with the help of code

**Comment by brianteeman** [comment]
~the code is there for you to read in the pr~

**Comment by brianteeman** [comment]
Sorry i was not thinking - just ignore me

**Comment by hardik-codes** [comment]
No issues

File: administrator/components/com_content/tmpl/article/edit.php [hunk 1]
```diff
@@ -51,6 +51,7 @@
 		<div class="row">
 			<div class="col-md-9">
 				<fieldset class="adminform">
+				    <?php echo $this->form->getLabel('articletext'); ?>
```
Comment (Quy) [inline administrator/components/com_content/tmpl/article/edit.php]:
```suggestion
					<?php echo $this->form->getLabel('articletext'); ?>
```

**Comment by hardik-codes** [comment]
@Quy I wanted to know why does this happen because the indentation seems to be fine in my text editor but after having pushed the code to github, the indentation is disturbed

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**